package.json

```diff
@@ -54,6 +51,8 @@
     "react-dom": "16.13.0",
     "react-test-renderer": "16.13.0",
     "rollup": "1.32.0",
-    "rollup-plugin-babel": "4.3.3"
+    "ts-jest": "25.2.1",
+    "tslib": "1.11.0",
+    "typescript": "3.8.2"
```

Comment:
I'd love to replace the entire rollup + typescript setup with just [tsdx](https://github.com/jaredpalmer/tsdx), so we don't have to deal with any ts configs, etc. Do you think it would make sense to update this in this PR?

Reply:
@glennreyes I just didn't get any experience with [tsdx](https://github.com/jaredpalmer/tsdx), but I can try it

Reply:
Okay cool! Let's open an issue at least if we want to move to it on a separate step.

Reply:
@glennreyes sounds good. I can open it when this PR wil be merged to be consistance